### DISCUSSION TITLE
"Unsaved disk"

### DISCUSSION THREAD

**Discussion by neleabels** [dirkwhoffmann/virtualc64#757] [Q&A]
I am feeling a bit stupid.

When I have made some changes to a disk and want to eject it or to replace it with another disk, the emulator warns me that the changes to the disk are not saved and will be lost if I proceed. Which they are.

The only way around this which I have found so far is to "export" the disk file to the same name.

Is this really the way it is supposed to be? That would be... clumsy, to say the least. What am I overlooking?

<img width="1280" alt="Bildschirm­foto 2023-01-29 um 14 15 47" src="https://user-images.githubusercontent.com/123799276/215328715-faec96a7-5210-4b2d-98ce-976c4f9a5af5.png">

**Comment by dirkwhoffmann** [comment]
It's supposed to be that way. I didn't want VirtualC64 to touch any files on the host machine. It doesn't even know about the filename where the disk originally came from.

**Comment by ha24-1** [comment]
... "didn't want VirtualC64 to touch any files on the host machine" - so maybe the export-function could have the ability to overwrite already  exported .d64 on the host? Yes, I understand: "no touch" includes existing .d64 ...
But to export any progress of a programming project as a new .d64 is not the "yellow from the egg" :-)
